README.rst

```diff
@@ -6,40 +6,56 @@ Testing/development buildouts for Plone
 
 This repository contains a number of configuration files for using
 `zc.buildout`_ to quickly set up a testing/development environment for your
-package.  The intended usage is to create a ``buildout.cfg`` like::
+package.
+
+A good example of how to use this, is `bobtemplates.plone <https://github.com/plone/bobtemplates.plone>`_.
+Follow the instructions there to create a Plone add-on, and look at the created files,
+especially the various buildout configs and `.travis.yml <https://github.com/plone/bobtemplates.plone/blob/master/bobtemplates/plone/addon/.travis.yml.bob>`_.
+
+The intended usage is to create a ``buildout.cfg`` like::
 
     [buildout]
-    extends = https://raw.github.com/collective/buildout.plonetest/master/test-4.x.cfg
+    extends = https://raw.github.com/collective/buildout.plonetest/master/test-5.x.cfg
     package-name = plone.app.foo
 
-Running buildout should give you a ``bin/test`` script, which can be used to
+Create a virtualenv and run buildout.
+This should give you a ``bin/test`` script, which can be used to
 run your package's tests.  If your tests have additional dependencies, you
 should declare them via the ``extras_require`` parameter of
 ``setuptools.setup`` and refer to the key used using the ``package-extras``
 variable::
 
     [buildout]
-    extends = https://raw.github.com/collective/buildout.plonetest/master/test-4.x.cfg
-    package-name = plone.app.foo
-    package-extras = [test]
-
-If you want to do `continuous integration`_ with `Travis CI`_ you should
-create a ``travis.cfg`` like::
-
-    [buildout]
-    extends = https://raw.github.com/collective/buildout.plonetest/master/travis-4.x.cfg
+    extends = https://raw.github.com/collective/buildout.plonetest/master/test-5.x.cfg
     package-name = plone.app.foo
     package-extras = [test]
 
+If you want to do `continuous integration`_ with `Travis CI`_ you can use this same buildout config.
 And a ``.travis.yml`` like::
 
     language: python
     python: 2.7
+    sudo: false
+    cache:
+      pip: true
+      directories:
+      - eggs
+    before_install:
+    - virtualenv -p `which python` .
+    - bin/pip install -r requirements.txt
+    - bin/buildout -N -t 3 annotate
```

Comment:
This last line shouldn't be here, should it? It's duplicate with the one in the `install` section just below.

Reply:
It is not the same. In `before_install` we call `bin/buildout annotate`, so the parsed configuration is printed for debug purposes, and in `install` it is the standard `bin/buildout` call.